gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/presenter/slots/ISingleSlot.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2014 ArcBees Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.gwtplatform.mvp.client.presenter.slots;
+
+import com.gwtplatform.mvp.client.PresenterWidget;
+
+/**
+ * The Interface ISingleSlot.
+ *
+ * @param <T> the generic type
+ */
+public interface ISingleSlot<T extends PresenterWidget<?>> extends ISlot<T> {
```

Comment:
We don't put "I" on interfaces but rather use more meaningful names. Maybe `BaseSingleSlot`, `SingleSlotDefinition`, etc.